iridium-core-server/src/main/java/software/iridium/api/TokenAuthenticationFilter.java

```diff
@@ -56,6 +58,13 @@ protected Object getPreAuthenticatedPrincipal(final HttpServletRequest httpServl
   private String extractBearerToken(final HttpServletRequest request) {
     final var header = request.getHeader(HttpHeaders.AUTHORIZATION);
 
+    if (header != null
+        && header.contains("Basic")
+        && request.getServletPath().equalsIgnoreCase(EXCHANGE_PATH)) {
+      // this is basic authentication
+      return null;
+    }
```

Comment:
This snippet bothers me.  It feels like Basic Authentication is called in error here and in my thinking we should log out or throw an error that lets the user know.  But I realize that null obeys the behavior of the interface in all other ways and agrees with the rest of the return values.

Reply:
Good call, @rwithers .  I felt like I was jamming this logic in incorrectly. 

I've update the PR and rolledback those changes to use a Spring convention.

```
  @Bean
  public WebSecurityCustomizer webSecurityCustomizer() {
    return (web) -> web.ignoring().requestMatchers("/oauth/token");
  }
```